lightning/src/routing/router.rs

```diff
@@ -772,7 +775,7 @@ impl Writeable for PaymentParameters {
 			(5, self.max_channel_saturation_power_of_half, required),
 			(6, self.expiry_time, option),
 			(7, self.previously_failed_channels, required_vec),
-			(8, *blinded_hints, optional_vec),
+			(8, blinded_hints, option),
```

Comment:
I think previously we were writing a length prefix for this, actually... Giving this a second look.

Reply:
Hm, the current code is passing a round trip ser test actually. I don't think there's a yes/no flag written for `option`s, and `optional_vec` writes as `WithoutLength`, IIUC, so it might be fine? Will expand the test to include multiple hints..